python_modules/libraries/dagster-components/dagster_components/core/component.py

```diff
@@ -114,13 +144,13 @@ def wrapper(actual_cls: Type[Component]) -> Type[Component]:
         return cls
 
 
-def is_component(cls: Type) -> bool:
+def is_registered_component(cls: Type) -> bool:
```

Comment:
why change name?

Reply:
Because it is possible for something to be a functioning `Component` without being marked for registration, which is what this predicate is checking for.